linux-create-vm.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Install git
+  echo "git should be installed"
+if ( which git )
+then 
+  echo "git already installed"
+else 
+  echo "installing git"
+  sudo apt install git
+fi
+
+#install nano
+  echo "nano should be installed"
+if ( which nano )
+then
+  echo "nano already installed"
+else
+  echo "installing nano"
+  sudo apt install nano 
+fi
+
+#install snap
+  echo "nano should be installed"
+if ( which snap )
+then
+  echo "snap already installed"
+else
+  echo "installing snap"
+  sudo apt install snap 
+fi
+
+# Install multipass
+  echo "installing multipass on $(uname)"
+if  ( which multipass )
+then
+  echo "multipass already installed on $(uname)"
+else
+  echo "installing multipass on $(uname)"  
+  sudo snap install multipass
+  sleep 5
+fi
+
+# Set Multipass local driver to Qemu
+#if [ "$(multipass get local.driver)" = "qemu" ]
+#then
+#  echo -e "\n==== Qemu local driver set ====\n"
+#else
+#  echo -e "\n==== Setting Qemu as Multipass local driver ====\n"
+#  multipass set local.driver=qemu
+#fi
+
+# ssh keys
+if [ -f "./id_ed25519" ]
+#if [ stat "./id_ed25519" ]
+then
+  echo "trifecta ssh key exists"
+else
+  echo "creating ssh keys for trifecta"
+  ssh-keygen -f ./id_ed25519 -t ed25519 -b 4096 -N ''
```

Comment:
does this key type take bit length?

Reply:
it can but it doesn't do anything, found that "ed25519 keys have a fixed key length of 256 bits"